spec/controllers/spree/admin/orders_controller_spec.rb

```diff
@@ -70,14 +70,13 @@ def self.make_simple_data!
         order_attributes.all?{ |attr| keys.include? attr }.should == true
       end
 
-      it "sorts orders in descending id order" do
+      it "sorts orders in ascending id order" do
```

Comment:
I found that this controller action was being used by the orders page, and two json endpoints. The orders page now doesn't need to load orders at all when loading the page template, and the other two endpoints actually order the data themselves (the orders page gets descending order as part of the pagination, and bulk order management reorders all the results as it's combining them with data from a line_items query). 

So as part of the refactor of the index action, I stripped out a lot of useless code including a few lines that explicitly reorder the items in the query, because actually both requests reorder the data themselves and it didn't seem needed any more. Both of those pages actually do show their items in descending order, but the json response itself doesn't. Does that make sense?

Reply:
I'll add some more specs this morning.

Reply:
That makes a lot of sense, thank you. If the order doesn't matter for our current consumers, should we remove the spec for ordering? It's obviously not a specification/requirement of the system.